modules/chapter2/pages/section4.adoc

```diff
@@ -0,0 +1,141 @@
+  - Clone the https://github.com/RedHatQuickCourses/rhods-qc-apps repository in this workbench.
+  - Execute the `rhods-qc-apps/1.intro/chapter2/lab/test_connection.ipynb` notebook in this workbench.
+
+* Create a workbench in `intro-projects-b`.
+  - Select any image for this workbench.
+  - Pass your name in the `STUDENT_NAME` environment variable.
+  - Clone the https://github.com/RedHatQuickCourses/rhods-qc-apps repository in this workbench.
+  - Execute the `rhods-qc-apps/1.intro/chapter2/lab/test_env_vars.ipynb` notebook in this workbench.
+
+* The `data-scientists` group must have edit permissions in `intro-projects-a`.
+
+* Only the `user4` user must have edit permissions in `intro-projects-b`.
+
+
+== Solution
+
+1. Create the projects.
+
+a. From your RHODS dashboard, click btn:[Data Science Projects] in the left navigation pane.
+
+b. In the btn:[Data science projects] page, click btn:[Create data science project].
+
+c. Enter `intro-projects-a` as the project name and click btn:[Create].
+
+d. Repeat the same steps to create `intro-projects-b`.
+
+
+2. Create a workbench connected to S3 in `intro-projects-a`.
+
+a. Navigate the `intro-projects-a` dashboard and click btn:[Create workbench].
+In the workbench creation page, enter the workbench name, select the workbench image, and the image version.
+
+b. Select btn:[Tiny] as the container size and 0 GPUs.
+
+c. In the btn:[Data connections] section, select btn:[Use a data connection].
+
+d. Enter the configuration values of your S3 connection.
+
+e. Click btn:[Create workbench] to finish the creation of the workbench and the data connection.
+
+
+3. Verify the connection of the workbench in `intro-projects-a`.
+
+a. Open the workbench of the `intro-projects-a` project.
+
+b. Log in to JupyterLab and accept the access form.
+
+c. Clone the https://github.com/RedHatQuickCourses/rhods-intro repository.
+
+d. Navigate to `rhods-qc-apps/1.intro/chapter2/lab` and run the `test_connection.ipynb` notebook.
+You should see the buckets available in S3.
+
+
+4. Create a workbench that receives the `STUDENT_NAME` environment variable in `intro-projects-b`.
+
+a. Navigate the `intro-projects-b` dashboard and click btn:[Create workbench].
+In the workbench creation page, enter the workbench name, select the workbench image, and the image version.
+
+b. Select btn:[Tiny] as the container size and 0 GPUs.
+
+c. In the btn:[Environment variables] section, click btn:[Add variable].
+
+d. Select btn:[Config Map] as the environment variable type.
+Then select btn:[Key/value].
+
+e. Enter `STUDENT_NAME` as the key, and your name as the value.
+
+f. Click btn:[Create workbench] to finish the creation of the workbench and the data connection.
+
+
+5. Verify the variable injected in the workbench in `intro-projects-b`.
+
+a. Open the workbench of the `intro-projects-b` project.
+
+b. Log in to JupyterLab and accept the access form.
+
+c. Clone the https://github.com/RedHatQuickCourses/rhods-intro repository.
```

Comment:
move to qc-apps repo?

Reply:
This is already in https://github.com/RedHatQuickCourses/rhods-qc-apps/pull/2 

I will adjust the repo URL.